packages/client/hmi-client/src/components/model/model-parts/tera-model-part-entry.vue

```diff
@@ -7,9 +7,10 @@
 				<tera-input-text v-else placeholder="Add a name" v-model="nameText" />
 			</span>
 			<span class="unit" :class="{ time: isTimePart }">
-				<template v-if="input || output">
-					<span><label>Input:</label> {{ input }}</span>
-					<span><label>Output:</label> {{ output }}</span>
+				<template v-if="subject || outcome">
+					<span><label>Subject:</label> {{ subject }}</span>
```

Comment:
Do we need to have {{ subject ?? "" }} to handle the case where we have an outcome therefore the above if passes, but we do not have a subject?

Same goes for the reverse scenario with having a subject but not having a controller

Reply:
If you meant the formatting I can take a run at improving it. 
If you meant the predicate check that is fine as is.

Reply:
There
<img width="654" alt="image" src="https://github.com/user-attachments/assets/caa61188-4c17-4a73-a2b9-70047b773107" />